style.css

```diff
@@ -648,7 +601,105 @@ textarea::placeholder {
 
 /* side menu  */
 
+/* popup  */
+.modal-container {
+  position: fixed;
+  height: 100vh;
+  width: 100vw;
+  backdrop-filter: blur(15px);
+  z-index: 100;
+  display: none;
+  justify-content: center;
+  align-items: center;
+
+  /* display: none; */
+}
+
+.modal {
+  width: auto;
+  max-width: 500px;
+  max-height: 700px;
+  background-color: #fff;
+  padding: 0 20px 20px 20px;
+  margin: 20px;
+  border-radius: 16px;
+  transform: scale(0);
+  transition: all ease-in-out 0.5s;
+  position: relative;
+}
+
+.open-modal {
+  transform: scale(1);
+  transition: all ease-in-out 0.5s;
+}
+
+.open-modal-container {
+  display: flex;
+}
+
+.modal-image img {
+  width: 100%;
+
+  /* max-height: 500px; */
+}
```

Comment:
- [ ] Please make sure you remove commented code from your project to make your code clean. :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)